config/jobs/kubernetes/sig-node/sig-node-presubmit.yaml

```diff
@@ -220,9 +220,10 @@ presubmits:
       containers:
       - image: gcr.io/k8s-testimages/kubekins-e2e:v20210601-ea6aa4e-master
         args:
-        - --repo=k8s.io/kubernetes=master
+        - "--repo=k8s.io/$(REPO_NAME)=$(PULL_REFS)"
```

Comment:
just to doublecheck: this is a more generic case of the former code we had, right? meaning: after your change we still can handle the previous case IIUC, so we're just more flexible, right?

Reply:
I suspect that this is causing the pre-submit to always run on master, not on the PR iteslf (but I am not that familiar with this stuff). If you look here, it runs on master: https://storage.googleapis.com/kubernetes-jenkins/logs/pull-kubernetes-node-kubelet-serial/1402355480164044800/build-log.txt

All the other pre-submits have this as well..

Reply:
Oops, I missed that, good catch.